### DISCUSSION TITLE
Bug in the dev_exp_previewer

### DISCUSSION THREAD

**Discussion by Mosexelle** [Monika-After-Story/MonikaModDev#8677] [QA - code and content creation (technical)] [answered]
Hi there!
I've got this error (copy from errors.txt):

I'm sorry, but errors were detected in your script. Please correct the
errors listed below, and try again.

File "game/dev/dev_exp_previewer.rpy", line 2299: Overly long logical line. (Check strings and parenthesis.)
(Perhaps you left out a ' at the end of the first line.)
                

Ren'Py Version: Ren'Py 6.99.12.4.2187

I attach this file if it helps.
[errors.txt](https://github.com/Monika-After-Story/MonikaModDev/files/8009120/errors.txt)
(Hopefully I didn't harm Monika, that's a relief!)

**Comment by multimokia** [comment, marked as answer]
Looks like it downloaded HTML. Did you download the `raw` file of the exp_previewer? That'd give you the proper rpy file of it

**Reply by Mosexelle** [reply]
I guess you're right. Damn, I will check better next time. Sorry for this.
